index.html

```diff
@@ -4,11 +4,195 @@
+      <section id="hero-section">
+        <div class="hero-wrapper">
+          <p class="hero-text-welcome">Hello world!</p>
+          <h1 class="hero-text-header">
+            THE INTERNATIONAL HACKATHON SUMMIT 2022
+          </h1>
+          <p class="hero-text">
+            Hackathons which help you put your coding skills to work. Solve
+            interesting business problems and real-world challenges. Register
+            now.
+          </p>
+          <h2 class="hero-text-time">2022.12.01(SUN) ~ 16(FRI)</h2>
+          <h3 class="hero-text-place">@ Exhibition gathering, Cairo, Egypt</h3>
+        </div>
+      </section>
+      <section id="program">
+        <div class="program-title">
+          <h2>You will learn</h2>
+          <div class="underline"></div>
+        </div>
+        <div class="program-cards">
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_01.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_02.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_03.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_04.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+          <div class="program-card">
+            <div class="img-box">
+              <img src="./assets/icons/program_icon_05.png" alt="icon" />
+              <h3>lorem</h3>
+            </div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni,
+              animi?
+            </p>
+          </div>
+        </div>
+        <a class="program-call-to-action" href="#">
+          Join Hackathon Global Summit 2022
+        </a>
+      </section>
+      <section id="speakers">
+        <div class="program-title">
+          <h2>Speakers</h2>
+          <div class="underline"></div>
+        </div>
+        <div class="chevron">
+          <div class="speakers-wrapper"></div>
+        </div>
```

Comment:
Kindly check and ensure that the speakers section conform to the design. The height of the text and title part are currently longer than the picture part which is not suppose to be so. Please rectify this anomaly :pray: 

Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-19 16-19-03](https://user-images.githubusercontent.com/65233860/196749773-0df1f19a-ca86-464d-b028-f7dc947cd8dc.png)  |  ![Screenshot from 2022-10-19 16-21-20](https://user-images.githubusercontent.com/65233860/196749813-90381dee-cad4-4724-b109-d666e319eecf.png)